rides/__init__.py

```diff
@@ -340,6 +341,11 @@ def join_ride(car_id, user):
             db.session.add(rider)
             db.session.add(car)
             db.session.commit()
+            u = User.query.get(username)
+            user_str = f"{u.firstname} {u.lastname}"
+            if not u[0].isdigit():
+                user_str += f"(@{username})"
```

Comment:
You probably want a space here, right?
```suggestion
                user_str += f" (@{username})"
```

Reply:
F strings don't need spaces as far as I am aware

Reply:
Without a space this will be: `Mary Strodl(@mstrodl)` rather than `Mary Strodl (@mstrodl)`

Reply:
Woops, thought the space was between the f and the "